packages/autorest.python/autorest/__init__.py

```diff
@@ -35,6 +36,7 @@ def __init__(self, *, output_folder: Union[str, Path], **kwargs: Any) -> None:
                 "Loading python.json file. This behavior will be depreacted"
             )
         self.options.update(python_json)
+        self._black = python_json.get("black", True)
```

Comment:
users can still turn off black

Reply:
@msyyc is there really no way to keep black as a separate plugin? Conceptually it really is a plugin, so it makes the code a lot less clean by adding black throughout the different plugins. I can take a look in a bit to see if it can't be done, but I think there should be a way to fix the current plugin we have, instead of doing this refactoring

Reply:
@iscai-msft It is nice if you can take a look. [stream.flush](https://github.com/Azure/autorest.python/blob/a9bb71d2ed41d033f3610e73042ded9d84350b0c/packages/autorest.python/autorest/jsonrpc/stdstream.py#L52) can't ensure flush to disk before black read it. The memory inconsistency is always hard to handle so I think the best way is to bypass it completely. Plugin `black` may be clean but it can't work for `multiapi/multiclient`. In one word, my fix pr is one of possible ways. 
add @lmazuel for awareness.